relevanceai/operations_new/ops.py

```diff
@@ -0,0 +1,166 @@
+from typing import Any, Dict, List, Optional
+from relevanceai.dataset.write import Write
+from datetime import datetime
+
+
+class Operations(Write):
+    def store_operation_metadata(self, operation: str, values: str):
+        """
+        Store metadata about operators
+        {
+            "_operationhistory_": {
+                "1-1-1-17-2-3": {
+                    "operation": "vector", "model_name": "miniLm"
+                },
+            }
+        }
+
+        """
+        print("Storing operation metadata...")
+        timestamp = str(datetime.now().timestamp()).replace(".", "-")
+        metadata = {
+            "_operationhistory_": {
+                timestamp: {"operation": operation, "parameters": values}
+            }
+        }
+        return self.upsert_metadata(metadata)
```

Comment:
this will overwrite previous _operationhistory_

Reply:
Great spot! Thanks - fixed